### DISCUSSION TITLE
Single cell RNAseq

### DISCUSSION THREAD

**Discussion by ggonnella** [egeulgen/pathfindR#193] [Q&A] [answered]
Can it be used with scRNA-seq data, eg. from Seurat? I was thinking of pseudobulk comparison of cell types in two conditions.

**Comment by egeulgen** [comment, marked as answer]
thanks for your interest @ggonnella. You can use pathfindR with any input data containing Gene symbols and associated significance values. For scRNAseq, you can first run [`Seurat::FindMarkers`](https://satijalab.org/seurat/reference/findmarkers) for finding differentially expressed genes and use the result as input for active-subnetwork search + enrichment analysis via `pathfindR`.
